sinatra-contrib/lib/sinatra/content_for.rb

```diff
@@ -128,9 +129,25 @@ module ContentFor
     #
     # Your blocks can also receive values, which are passed to them
     # by <tt>yield_content</tt>
-    def content_for(key, value = nil, &block)
+    def content_for(key, *args, &block)
```

Comment:
I would rather see the method signature:

`def content_for(key, value = nil, options = {}, &block)`

It would avoid the need to jump through the hoops below.